estate/models/estate_property.py

```diff
@@ -0,0 +1,43 @@
+from odoo import fields, models
+
+class EstateProperty(models.Model):
+    _name = "estate_property"
+    _description = "Real estate management"
+
+    active = True
```

Comment:
what is it for ?
active is a "magical" field in odoo. it should be a real field (not a property)
so `active = fields.Boolean("Is Active?", default=T/F)`
And if False, record will not be returned by the ORM during search or accessing related model records.